images/linux/scripts/installers/git.sh

```diff
@@ -52,3 +52,7 @@ else
     echo "[!] Hub CLI was not installed"
     exit 1
 fi
+
+# Add well-known SSH host keys to known_hosts
+ssh-keyscan -t rsa github.com >> /etc/ssh/ssh_known_hosts
+ssh-keyscan -t rsa ssh.dev.azure.com >> /etc/ssh/ssh_known_hosts
```

Comment:
I suggest to put this logic to `https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/installers/configure-environment.sh`

Reply:
Discussed offline. Decided to leave here